### PR TITLE
Create build.yml

### PR DESCRIPTION
**Pull request by Bisi0n**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: .NET
```
Comment (md-jafari) [inline .github/workflows/build.yml:1]: Use a better name that explains the workflow.

**Review by md-jafari — CHANGES_REQUESTED** [review]
One small issue.

File: .github/workflows/build.yml (hunk 1 above)
Comment (Bisi0n) [reply .github/workflows/build.yml]: fixed :)

**Review by md-jafari — APPROVED** [review]
We have a better name now!

**Pull request merged**